src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -20,11 +17,14 @@ public class ZipkinClient : ITracerClient
         private string requestName;
         private ITraceProvider traceProvider;
         private ILog logger;
+        private List<string> internalDomainList;
 
         public ZipkinClient(ITraceProvider tracerProvider, string requestName, ILog logger) : this(tracerProvider, requestName, logger, new ZipkinConfig(), new SpanCollectorBuilder()) { }
 
         public ZipkinClient(ITraceProvider traceProvider, string requestName, ILog logger, IZipkinConfig zipkinConfig, ISpanCollectorBuilder spanCollectorBuilder)
         {
+            internalDomainList = zipkinConfig.GetInternalDomainList();
```

Comment:
The ruby opensource (we do not control it) implementation already does that. 
If we do not do the same in C# we will have both a full url and a truncated URL sent to Zipkin and searches, etc all will look double